nullaway/src/test/java/com/uber/nullaway/NullAwayTest.java

```diff
@@ -931,4 +940,24 @@ public void testCastToNonNull() {
             "}")
         .doTest();
   }
+
+  @Test
+  public void testReadStaticInConstructor() {
+    compilationHelper
+        .addSourceLines(
+            "Test.java",
+            "package com.uber;",
+            "import javax.annotation.Nullable;",
+            "class Test {",
+            "  // BUG: Diagnostic contains: @NonNull static field o not initialized",
```

Comment:
So, before the error showed up in the constructor?

Reply:
No, before we crashed due to [this assertion](https://github.com/uber/NullAway/blob/d8dc8f05defd9609a95113c707dbd1ed44d8e02b/nullaway/src/main/java/com/uber/nullaway/dataflow/AccessPathNullnessAnalysis.java#L174) failing 😄